### ISSUE TITLE
Make unread counter in title opt-in

### ISSUE THREAD

**Issue by trwnh** [tootsuite/mastodon#12211]
#11560  added a counter of unread notifications to the page title. This behavior should be opt-in.

**Comment by joyeusenoelle** [comment]
A note in support of this. As I said [here](https://elekk.xyz/@noelle/103026101559005567), when I first joined Mastodon in April 2017 having "unread notifications" in the window title was considered an antipattern and many users were vocal about appreciating its absence. This behavior should be opt-out at *minimum* and optimally should be opt-in.

**Comment by QuarkTheAwesome** [comment]
+1 for having this be optional. I find it extremely difficult to ignore counters from certain apps/websites, and while I can tough it out it, it's much easier for me to manage where my attention is going if I can disable this kind of thing.

**Comment by nsuchy** [comment]
Coming from sites like Facebook and Twitter, I would expect this behavior? Perhaps make it opt-out, or ask them in the user onboarding process (the welcome to Mastodon tutorial)

**Comment by trwnh** [comment]
>Coming from sites like Facebook and Twitter, I would expect this behavior?

I would argue that Mastodon should not be like Facebook or Twitter. Those sites also have algorithmic feeds and advertisements. Expected behavior of other products shouldn't be copied blindly.

**Comment by nsuchy** [comment]
I agree we shouldn’t copy blindly. We should consider what philosophical trade offs we are willing to make to increase adoption.

**Comment by wolfpurplemoon** [comment]
When it first started it was very unclear what it was notifying about, in a multi column layout it could be something new in any of the columns. If I want to be instantly notified about notifications there is already the nice bloop sound

**Comment by harrybailey** [comment]
Looks like since 2019 this chance has both been merged / added and then removed again.

So this issues would now be a suggestion, or even closed.
